### PR TITLE
ZEN-26937 Display full class path in device banner

### PR DESCRIPTION
**Pull request by karolzen**
https://jira.zenoss.com/browse/ZEN-26937

### CODE REVIEW

File: Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js [hunk 1]
```diff
@@ -1408,12 +1408,45 @@ Ext.define("Zenoss.InfraDetailNav", {
...
+        titlePosition: 0,
+        items: [{
+                   xtype:'displayfield',
+                   value: _t('/'),
+                   width: 'auto',
+                   cls: 'x-device-panel-header',
+                   id: 'device_panel_header',
+                   allowBlank : true,
+                   readOnly: true
+               }]
```
Comment (jpeacock-zenoss) [inline Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js]: Please deindent this by two levels.  It doesn't need to be pushed so far to the right.  E.g.
```
    items: [{
        xtype:'displayfield',
        value: _t('/'),
        width: 'auto',
        cls: 'x-device-panel-header',
        id: 'device_panel_header',
        allowBlank : true,
        readOnly: true
    }]
```

File: Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js [hunk 2]
```diff
@@ -1408,12 +1408,45 @@ Ext.define("Zenoss.InfraDetailNav", {
     }
 });
 
+var truncate = function(cmp_name, width, uid) {
+    var path = uid.replace('/zport/dmd/', ''),
```
Comment (jpeacock-zenoss) [inline Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js]: In 4.x, where the full path was visible, the path started with a `/`.  You can see this in the screenshots shown in the ticket.

To preserve this, the `replace` should be rewritten as `uid.replace('/zport/dmd', '')`.

File: Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js (hunk 1 above)
Comment (karolzen) [inline Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js]: Ok :+1:

File: Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js (hunk 2 above)
Comment (karolzen) [inline Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js]: Ok, I will change it to 4.x format.

**Pull request merged**